src/epub/css/local.css

```diff
@@ -41,10 +46,6 @@ section[epub|type~="dedication"] b{
 	text-indent: 0;
 }
 
-[epub|type~="z3998:poem"] p + p{
```

Comment:
If this selector is unused, why didn't lint catch it? Are you sure it's unused?

Reply:
Actually, lint did catch it. This CSS became unused after I tweaked the poems in b19df3f.